libraries/standard/mqtt/include/iot_mqtt.h

```diff
@@ -325,7 +323,7 @@ void IotMqtt_Disconnect( IotMqttConnection_t mqttConnection,
  * subscriptions.
  * @param[in] subscriptionCount The number of elements in pSubscriptionList.
  * @param[in] flags Flags which modify the behavior of this function. See @ref mqtt_constants_flags.
- * @param[in] pCallbackInfo Asynchronous notification of this function's completion.
+ * @param[in] pCallbackInfo Asynchronous notification of this function's completion (NULL to disable).
```

Comment:
Line 811 already says something similar:

> Pass NULL to ignore

For consistency, either change the "NULL to disable to that" or change that line to "NULL to disable".

(We will need to propagate this change to other libraries)